modules/setup/setup.js

```diff
@@ -142,9 +132,11 @@ class Setup {
                 if (_stdout !== `777 ${__dirname}/../../config.json`) {
                     child_process.exec(`sudo chmod -R 777 ${__dirname}/../../config.json`);
                 }
-            } else {
-                Logger.verbose(`Error in check permission while starting nodes _stderr: ${_stderr}`);
-                Logger.verbose(`Error in check permission while pulling docker image: ${error}`);
+            } else if(error){
+                Helper.errorLogHandler(`Error while changing config.json file permission: ${error}`, 'Error occured while setting up node due to permission issue');
+            } 
+            else {
+                Helper.errorLogHandler(`Error while changing config.json file permission: ${_stderr}`, 'Error occured while setting up node due to permission issue');
```

Comment:
This both code blocks are exactly same. 
It can be simplified as 
error = error || _strderr . Also dont use underscore in variable names as our codebase doesn't follow that style.